docs/Operators.md

```diff
@@ -127,7 +127,7 @@
   * **input**:
     <dl>
       <dt>X</dt>
-      <dd>Input data tensor from the previous operator; dimensions for image case are (N x C x H x W), where N is the batch size, C is the number of channels, and H and W are the height and the width of the data. For non image case, the dimension are in the form of (N x C x D1 x D2 ... Dn), where N is the batch size.</dd>
```

Comment:
This doesn't match the current `AveragePool` doc at https://github.com/onnx/onnx/blob/master/onnx/defs/nn/defs.cc#L35. Is it possible your working copy is outdated? Note after changing C++ files you need to reinstall in order to get the changes effective.

Reply:
This sounds like a good reason to move the spec files out of C++ :)

Reply:
On the other hand you want to have these operators schema in c++ for being able to do validation on any environment. :-)

Reply:
which files do I need to re-install/what is being pulled in exactly here?

Reply:
How did you install onnx at the beginning? The procedure of changing docs in `Operators.md` is first update the `*.defs.cc` file, and then install the new sources into your python environment (there is a "Source" section in our repo frontpage telling you how to install from source) and then run `python onnx/defs/gen_doc.py` to get the new `Operators.md` file.